_guides/kubernetes.adoc

```diff
@@ -166,7 +166,7 @@ The Docker registry can be specified with the following property:
 
 [source]
 ----
-quarkus.container-image.registry=http://my.docker-registry.net
+quarkus.container-image.registry=my.docker-registry.net
```

Comment:
@geoand didn't we already fixed something similar somewhere? I have a vague recollection of it.